### PR TITLE
Added AWS Account resource

### PR DESCRIPTION
**Pull request by droyad**
Rough cut of adding the AWS Account as an alternative to #80

### CODE REVIEW

File: octopusdeploy/provider.go [hunk 1]
```diff
@@ -38,6 +38,8 @@ func Provider() terraform.ResourceProvider {
 			"octopusdeploy_channel":                           resourceChannel(),
 			"octopusdeploy_nuget_feed":                        resourceNugetFeed(),
 			"octopusdeploy_azure_service_principal":           resourceAzureServicePrincipal(),
+			"octopusdeploy_aws_account":           			   resourceAmazonWebServicesAccount(),
+			"octopusdeploy_aws_account":                       resourceAmazonWebServicesAccount(),
```
Comment (droyad) [inline octopusdeploy/provider.go:42]: Duplicate

File: octopusdeploy/resource_aws_account.go [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+			},
+			"tenanted_deployment_participation": getTenantedDeploymentSchema(),
+			"secret_key": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"access_key": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
```
Comment (droyad) [inline octopusdeploy/resource_aws_account.go:53]: 👍 Matches https://github.com/OctopusDeploy/Sashimi.Aws.Accounts/blob/main/source/Sashimi/AmazonWebServicesAccountResource.cs

**Review by droyad — COMMENTED** [review]
LGTM

**Comment by AdminTurnedDevOps** [comment]
@droyad Removed duplicate here. Thanks for the catch.

**Pull request merged**